src/reconstruction/ppmc_cuda_tests.cu

```diff
@@ -24,6 +24,8 @@
 
 TEST(tHYDROPpmcCTUReconstructor, CorrectInputExpectCorrectOutput)
 {
+  // Alwin: skip until this has been fixed
+  GTEST_SKIP();
```

Comment:
I'm happy to help fix this test but I don't think that just marking tests as skippable when we change things is a good idea.

Reply:
Marking a test as skippable when the test passed an incorrect function and fails the correct function is a good idea

Reply:
Fixed in [11804ba](https://github.com/cholla-hydro/cholla/pull/315/commits/11804ba2362387f67143ca0f1f135b1a69856a8d)